README.md

```diff
@@ -1 +1,179 @@
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📊 Daily Crypto <a name="about-project"></a>
+
+**CryptoNike** Is a website that provides users with the standing of Crypto currencies with their latest statistics accumulated during the 2019-2020 season
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://tailwindcss.com/docs/guides/create-react-app">Tailwind Css</a></li>
+    <li><a href="https://www.css3.com/">CSS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **List of coins with current market cap**
+- **Statistics of each coin** 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://react-capstone-liard.vercel.app/
+)
+
+- [video walkthrough Link](https://www.loom.com/share/e8dd0915a33f4564a0ffcd64121abe29)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+- Clone this repo as described in the setup section. 
+- Make modifications as preferred
+
+### Prerequisites
+
+In order to run this project you need: To clone or fork and run it in a browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/olanikegloria/CryptoNike.git
+```
+
+### Install
+
+- Run ` cd folder `
+- Run ` npm install `
+
+### Usage
+
+To run the project, execute the following command:
+
+- Run `npm start`
+
+### Run tests
+
+To run tests, run the following command:
+
+- to test validation errors `npx hint .`
+
+- to test CSS linter errors `npx stylelint "**/*.{css,scss}"`
+
+- to test JavaScript linter errors  `npx eslint .`
+
+- to run the jest tests `npx test`
+
+**Highlight**
+- My highlight for this app I have created is that I love the user interface
+- also the fact that I implemented a search bar
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **OLANIKE OLATUNJI**
+
+- GitHub: [@DjibrilM](https://github.com/DjibrilM)
+- LinkedIn: [ODjibrilM](https://www.linkedin.com/in/djibril-mugisho/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add charts to display the statistics of each coin
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/olanikegloria/CryptoNike/issues).
```

Comment:
- [ ] Kindly make sure the link you provoided is not broken

Reply:
This change has been done well